newIDE/app/src/GameDashboard/Feedbacks/FeedbackCard.js

```diff
@@ -58,61 +58,35 @@ const getRatings = (ratings: ?GameRatings) => {
   }
 };
 
-const getCurrentState = (pendingState: ?boolean, state: boolean) => {
-  return pendingState === null ? state : pendingState;
-};
-
 const FeedbackCard = ({ comment, authenticatedUser }: Props) => {
   const { getAuthorizationHeader, profile } = authenticatedUser;
   const ratings = getRatings(comment.ratings);
   const theme = React.useContext(GDevelopThemeContext);
 
-  const [processed, setProcessed] = React.useState<boolean>(
-    !!comment.processedAt
-  );
-  const [
-    pendingProcessedState,
-    setPendingProcessedState,
-  ] = React.useState<?boolean>(null);
+  const processComment = async (processed: boolean, i18n: I18nType) => {
+    if (!profile) return;
+    try {
+      await updateComment(getAuthorizationHeader, profile.id, {
+        gameId: comment.gameId,
+        commentId: comment.id,
+        processed,
+      });
```

Comment:
as discussed, you probably need to:
1. retrieve the comment returned from the call `const updatedComment = await udpateComment(...)`
2. call a `onCommentUpdated(updatedComment)` defined in the top component, which replaces the comment in question (findable by ID) by this one.

This would fix the sort applied on the correct updated list.